docs/reference/datatiers.asciidoc

```diff
@@ -62,51 +62,51 @@ hot tier.
 [[warm-tier]]
 === Warm tier
 
-Time series data can move to the warm tier once it is being queried less frequently 
-than the recently-indexed data in the hot tier. 
-The warm tier typically holds data from recent weeks. 
+Time series data can move to the warm tier once it is being queried less frequently
+than the recently-indexed data in the hot tier.
+The warm tier typically holds data from recent weeks.
 Updates are still allowed, but likely infrequent.
-Nodes in the warm tier generally don't need to be as fast as those in the hot tier. 
+Nodes in the warm tier generally don't need to be as fast as those in the hot tier.
 For resiliency, indices in the warm tier should be configured to use one or more replicas.
 
 [discrete]
 [[cold-tier]]
 === Cold tier
 
-Once data in the warm tier is no longer being updated, it can move to the cold tier. 
-The cold tier typically holds the data from recent months or years.
+Once data in the warm tier is no longer being updated, it can move to the cold tier.
```

Comment:
You're using "cost effective" as a euphemism for "slow", but we haven't really talked about cost elsewhere in this doc so it feels a bit awkward. Maybe we should say something in the introduction about why you might want tiers based on cost?

Possible alternative, with a bit less repetition:

```suggestion
Once data is no longer being updated, it can move from the warm tier to the cold tier where it stays
for the rest of its life.
```

Reply:
Thanks, that is more clear, I pushed 93683f74200ff2b0b33ec88899fd5ef74bcb2ded